frontend/src/Components/Team/index.js

```diff
@@ -1,12 +1,86 @@
-import React from "react";
-import { Typography } from "antd";
+import React, { useEffect, useState } from "react";
+import { Typography, Card, Table } from "antd";
 import { useParams } from "react-router-dom";
+import "./index.css";
 
-const Team = () => {
+import { Api } from "../../Api";
+
+const Team = (props) => {
   const { Title } = Typography;
   const { teamid } = useParams();
+  const [teamdata, setTeamdata] = useState(null);
+  const [render, setRender] = useState(<Title>Team: {teamid}</Title>);
+  const { Column, ColumnGroup } = Table;
+  const { Meta } = Card;
+
+  // Get team data
+  useEffect(() => {
+    if (props.teamdata) setTeamdata(props.teamdata);
+    else if (teamid) {
+      Api.get("/teams/" + teamid)
```

Comment:
Try `react-query` to improve fetching pattern: https://github.com/tannerlinsley/react-query. It's a util library that gives you nice way to handle any type of http request and handle it's `data`, `error` and `loading` state. Based on those values your component logic would be simpler.